CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 * plot_ppc animation: improve docs and error handling (#1162)
 
 ### Deprecation
+* `credible_interval` argument replaced by `hpd_interval`throughout with exception of `plot_loo_pit`
```

Comment:
there should be also a note on the difference in behaviour of `plot_posterior` so users know they have to change `None` to `"hide"`